test-framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -161,6 +161,25 @@ public static void restoreContentType() {
         Requests.INDEX_CONTENT_TYPE = XContentType.JSON;
     }
 
+    private static boolean TESTS_SEED_PROPERTY_SET = false;
+    @BeforeClass
+    @SuppressForbidden(reason = "sets tests.seed")
```

Comment:
> don't change tests.seed, i dont care what intellij does, this is wrong to do.

@rmuir Do you have another proposal on how we can ensure that tests are reproducible in IntelliJ when using the JUnit integration (I think having that is incredible important)? And why is this wrong?